name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/INGDiBaPDFExtractor.java

```diff
@@ -214,12 +214,17 @@ private void addDividendeTransaction()
                         t.setShares(asShares(v.get("shares")));
                     }
                 })
+                
+                // Ex-Dividenden-Tag 29.11.2016
```

Comment:
// Ex-Dividenden-Tag 29.11.2016
not the same for regEx
`^Ex-Tag (?<exDate>\\d+.\\d+.\\d{4})$`

Reply:
changed